src/mmw/js/src/core/templates/layerControlContainer.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <div class="layer-control-button-container leaflet-control leaflet-bar">
+        <a class="leaflet-bar-part leaflet-bar-part-single" href="#" title="Show layer control">
+            <span class="fa fa-globe"></span>
```

Comment:
I couldn't find it in the icon set we are using (Font Awesome). I'll file an issue.